config/api.yml

```diff
@@ -2280,6 +2280,8 @@
       - :name: read
         :identifier: physical_storage_show_list
       :post:
+      - :name: create_new_physical_storage
+        :identifier: create_new_physical_storage
```

Comment:
probably `physical_storage_new` as declared in ManageIQ's db/fixtures/miq_product_features.yml